gt-scheduler/website
# Fix delete display message

### Summary

Resolves #181 

Fixes UI issue where "\<unknown\>" is briefly displayed as delete modal transitions out.

### Checklist

- [x] “Are you sure you want to delete schedule <unknown>?” message is investigated and removed.
- [x] Popup should be closed immediately with no further UI changes.

### How to Test
Delete a schedule and check to see if "\<unknown\>" is briefly displayed as the modal transitions out.
